examples/basic/src/main.rs

```diff
@@ -1,109 +0,0 @@
-use opentelemetry::global::shutdown_tracer_provider;
```

Comment:
nit: it looks like a some cleanups are being done in this PR.. would be nice to separate them out, so the PR is more manageable in size . Github is struggling to show huge PRs for me 🤣

Reply:
Yeah it's large but would be challenging to split out as the pieces depend on each other. There isn't really a clean bridge from the previous sdk to this sdk. This example was not needed as it's demonstrating deprecated exporters now (jaeger).